### PR TITLE
Supplier update

### PR DESCRIPTION
**Pull request by Kushan1994**

### CODE REVIEW

File: app/Actions/GenarateBarcode.php [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<?php
```
Comment (sureshhemal) [inline app/Actions/GenarateBarcode.php:1]: Delete this file. `GenarateBarcode.php`

File: resources/js/Components/Supplier/Form.vue [hunk 2]
```diff
@@ -1,28 +1,52 @@
 <template>
     <FormGruop>
```
Comment (sureshhemal) [inline resources/js/Components/Supplier/Form.vue:2]: Replace `FormGruop` with correct spellings

File: resources/js/Components/Supplier/Form.vue [hunk 3]
```diff
@@ -33,6 +57,11 @@
             id="note"
             type="text"
         />
+        <div
+            v-if="form.errors.notes"
+            class="text-red-500 text-xs mt-1"
+            v-text="form.errors.notes"
+        ></div>
     </FormGruop>
```
Comment (sureshhemal) [inline resources/js/Components/Supplier/Form.vue:65]: where is the `textarea` field

File: resources/js/Components/Supplier/Form.vue [hunk 4]
```diff
@@ -46,22 +75,33 @@ import WebsolButton from "@/Components/NewButton";
...
+        type: String,
+        required: true,
+    },
+    method: {
+        type: String,
+        required: true,
+    },
+});
+let form = useForm({ ...props.supplier });
+const editableUUID = computed(() => {
```
Comment (sureshhemal) [inline resources/js/Components/Supplier/Form.vue:96]: add a space line between two separate sections

File: resources/js/Pages/Supplier/Index.vue [hunk 5]
```diff
@@ -1,5 +1,10 @@
 <template>
     <FrontLayout>
+        <div class="mb-6">
+            <InertiaLink :href="route('suppliers.create')">
```
Comment (sureshhemal) [inline resources/js/Pages/Supplier/Index.vue:4]: remove `:href=...` and make `@click=create` and `function create()`

File: resources/js/Pages/Supplier/Update.vue [hunk 6]
```diff
@@ -1,10 +1,24 @@
...
+            :method="`put`"
+            :url="props.url"
+        />
     </FrontLayout>
 </template>
 
 <script setup>
 import FrontLayout from "@/Layouts/FrontLayout";
 import SupplierForm from "@/Components/Supplier/Form";
+let props = defineProps({
```
Comment (sureshhemal) [inline resources/js/Pages/Supplier/Update.vue:14]: add a space line

**Review by sureshhemal — CHANGES_REQUESTED** [review]

File: resources/js/Pages/Supplier/Index.vue (hunk 5 above)
Comment (Kushan1994) [reply resources/js/Pages/Supplier/Index.vue:4]: later

**Review by sureshhemal — APPROVED** [review]

**Pull request merged**